Karamelfile

```diff
@@ -28,6 +28,13 @@ dependencies:
       - kube-hops::addons      
       - hops::docker_registry
 
+  - recipe: kube-hops::filebeat
+    global:
+      - kube-hops::node
+      - kube-hops::master
+      - kube-hops::addons      
+      - hops::docker_registry
```

Comment:
As you include the recipe in another recipe (correctly), Karamel is not aware of the existence of the filebeat recipe. Hence you don't need this part.

Reply:
fixed